blt/blt.yml

```diff
@@ -1,4 +1,4 @@
 project:
   machine_name: BLTed
   profile:
-    name: lightning
+    name: minimal
```

Comment:
Should we go for the `standard` profile by default? The `minimal` profile would be too bare bones for a casual user.

Reply:
BLT has itself switched to minimal: https://github.com/acquia/blt/issues/4244#issuecomment-696389383